src/components/organisms/header/header.html

```diff
@@ -0,0 +1,81 @@
+<header class="hearder_wrapper">
```

Comment:
```suggestion
<header class="header-wrapper">
```
クラス名は原則としてスネークケース(_でつなげる)ではなくケバブケース(-でつなげる)でお願いします。